api/geth_backend.go

```diff
@@ -692,6 +696,88 @@ func (b *GethStatusBackend) ConvertToKeycardAccount(account multiaccounts.Accoun
 	return nil
 }
 
+func (b *GethStatusBackend) CreateAccountAndLogin(request *requests.CreateAccount) error {
+
+	if err := request.Validate(); err != nil {
+		return err
+	}
+
+	if err := b.accountManager.InitKeystore(filepath.Join(request.BackupDisabledDataDir, "keystore")); err != nil {
+		return err
+	}
+
+	b.UpdateRootDataDir(request.BackupDisabledDataDir)
+	err := b.OpenAccounts()
+	if err != nil {
+		b.log.Error("failed open accounts", err)
+		return err
+	}
+
+	generator := b.accountManager.AccountsGenerator()
+
+	generatedAccountInfos, err := generator.Generate(12, 1, "")
```

Comment:
what do these numbers mean? perhaps a variable for context would be good? 🤔

Reply:
indeed :)